core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino

```diff
@@ -37,37 +37,47 @@ CONFIGURATION=(
     --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}"
 )
 
+declare -A CONFIG_ENV
 # Don't need shellcheck to follow env.sh
 # shellcheck disable=1091
 source /etc/trino/env.sh
 
+env_options=()
+if [ -z "${JAVA_HOME}" ]; then
+    # Don't expand JAVA_HOME in the error message
+    # shellcheck disable=2016
+    echo 'Warning: No value found for $JAVA_HOME. Default Java will be used.' >&2
+else
+    env_options+=("PATH=${JAVA_HOME}/bin:$PATH")
+fi
+sudo_options=(-u "$SERVICE_USER")
+for key in "${!CONFIG_ENV[@]}"; do
+    env_options+=("$key=${CONFIG_ENV[$key]}")
+done
+if [ "${#env_options[@]}" -ne 0 ]; then
+    sudo_options+=(env "${env_options[@]}")
+fi
```

Comment:
Sorry for back/forth, i got confused by the man page. the `env` is not needed, since `sudo` handles env setting internall, as used to use it to set PATH here

https://github.com/trinodb/trino/blob/cf4845d3316d8745ae9c34db4d356bc3827b6a5d/core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino#L41

```suggestion
```

Reply:
Np, done!